### PR TITLE
body-project

### PR DESCRIPTION
**Pull request by yegonkimutai**
A continuation of the Portfolio project. The task included adding the contents of the profile as well as the about page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -51,6 +57,143 @@ <h3 class="intro">I'm A Software Developer</h3>
...
+                <h3 class="header-2">Profesional Art Printing Data</h3>
+                <p class="p-2">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                </p>
+                <ul class="link-1">
+                    <li class="link--">html</li>
+                    <li class="link--">boostrap</li>
+                    <li class="link--">Ruby</li>
+                </ul>
+                <h3 class="footer">See Project</h3>
```
Comment (roshan-bajgain) [inline index.html:106]: - [ ] Your card is overlapping with the About section please fix it and also make the design similar to the Figma design.
- [ ] Also implement for all cards.
### Your design
his


### Figma design

fimga

File: index.html [hunk 2]
```diff
@@ -51,6 +57,143 @@ <h3 class="intro">I'm A Software Developer</h3>
...
+                        <ul ul class="fm">
+                            <li>Codekit</li>
+                            <li>Github</li>
+                            <li>Codepen</li>
+                            <li>Gitlab</li>
+                            <li>Terminal</li>
+                        </ul>
+                    </div>
+                </div>
+        </section>
```
Comment (roshan-bajgain) [inline index.html:197]: - [ ] You must stick to the design as much as possible like color, font, size, and background. Your design is overlapping so I would suggest you please fix this issue and also make every design of the `About section` similar to the Figma design. See the design below for reference.
### Your design
your

### Figma design
fimga

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @yegonkimutai, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -155,4 +437,34 @@ image {
   .p {
     width: 60%;
   }
+
+  .page-1 {
+    display: grid;
+    gap: 30px;
+    grid-template-columns: 100%;
+    grid-template-rows: 100%;
+    grid-template-areas:
```
Comment (wayungi) [inline style.css:446]: - [ ] Please fix the excess white space under your **My recent work**. Refer to this [document](https://cssgridgarden.com/) to test out your CSS grid syntax before applying it to the project.
_**Kindly remove this space**_
speace-1

File: index.html [hunk 4]
```diff
@@ -51,6 +57,139 @@ <h3 class="intro">I'm A Software Developer</h3>
...
+                    <li class="link--">boostrap</li>
+                    <li class="link--">Ruby</li>
+                </ul>
+                <button class="footer" >See Project</button>
+            </div>
+            <div class="grid-3">
+                <h3 class="header-2">Profesional Art Printing Data</h3>
+                <p class="p-2">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                </p>
```
Comment (wayungi) [inline index.html:95]: - [ ] As requested by the previous reviewer, Please fix your design to match the Figma template. Your content is currently overlapping.

_**figman**_
des-1


_**Your design**_
des-2

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @yegonkimutai , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
image

 ## Highlights
- [ ] Good PR title 👍🏾 
- [ ] GitHub flow 👏🏾 
- [ ] Descriptive commit message 😄 
- [ ] Professional README 💯 
- [ ] All linter checks passing ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._


 ## Required Changes ♻️
- [ ]  I noticed that you made very few changes recommended by the previous reviewer. To go to the next project and build your technical skills, you need to implement all the requested changes that are not [OPTIONAL]. Please open the Figma template and your project in the browser. Fire up your editor and fix up the issues one by one. Reach out to me on slack if anything is not clear. I will be happy to help.
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (yegonkimutai) [reply index.html:95]: Hello @wayungi , please explain to me why my project is overlapping in your screen since in my screen i cant see the overlapping

Comment (RiyaBulia12) [inline index.html:95]: Good job on fixing the overlapping issue. 🥳 
Screenshot 2023-01-21 at 12 30 18 PM

**Review by RiyaBulia12 — APPROVED** [review]
## STATUS: APPROVED 🏆🥇 

### Hi @yegonkimutai  👋🏽  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
#### Congratulations! 🎉 🎉 🥳 🥳 



## Highlights
* No Linter errors ✔️
* GitHub flow used ✔️
* Fixed the Previous CR comments ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**